backend/src/layer/data/flickr_api/api_request.rs

```diff
@@ -120,7 +120,7 @@ mod test {
 
     fn get_api_key() -> String {
         dotenv().ok();
-        env::var("FLICKR_PUBLIC_KEY").expect("FLICKR_PUBLIC_KEY should be set in the .env!")
+        env::var("FLICKR_API_KEY").expect("FLICKR_API_KEY should be set in the .env!")
```

Comment:
You might need to change this in the tests as well

Reply:
do you mean the test in github?

Reply:
No the ones from @Whatsuup

Reply:
aren't these the tests?